src/org/ice1000/julia/lang/action/julia-repl-actions.kt

```diff
@@ -73,14 +75,74 @@ class JuliaIncludeRunFileToReplAction : JuliaSendCodeToReplAction(
 class JuliaSendSelectionToReplAction : JuliaSendCodeToReplAction(
 	JuliaBundle.message("julia.actions.repl.run-selected.name"),
 	JuliaBundle.message("julia.actions.repl.run-selected.description")), DumbAware {
+
+
```

Comment:
Could you remove at least one empty line here?

Reply:
Sure.. done!